challenges/README.md

```diff
@@ -93,3 +125,22 @@ Response
     "is_user_subscribed":1
 }
 ```
+- **Error Response:**
+  - **Code:** 404
+    - **Content:** 
+    ```
+    {
+        "statusCode":404,
+        "error":"Not Found",
+        "message":"User cannot be subscribed to challenge"
```

Comment:
A space after the colon please
```json
{
  "statusCode": 404,
  "error": "Not Found",
  "message": "User cannot be subscribed to challenge"
}
```

Reply:
Syntax highlighting for all the code blocks will be better,
https://www.markdownguide.org/extended-syntax/#syntax-highlighting

Reply:
Changes commited